src/hooks/useAuthForm.tsx

```diff
@@ -18,17 +24,71 @@ export interface CommonFieldValues extends FieldValues {
   passwordConfirmation: string;
 }
 const useAuthForm = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
```

Comment:
In general have no idea what is going on there.
This login check should works at the app initialization